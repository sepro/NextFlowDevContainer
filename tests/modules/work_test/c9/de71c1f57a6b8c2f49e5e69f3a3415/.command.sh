#!/bin/bash

# Create output directory if it doesn't exist
mkdir -p test_output

# Create files with random names
for i in $(seq 1 3); do
    # Generate random filename
    random_name=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 12 | head -n 1)
    filename="file_${random_name}.txt"

    # Create the file with some content
    echo "This is file $i with random name: $filename" > "$filename"
    echo "Created at: $(date)" >> "$filename"
    echo "Random content: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 50 | head -n 1)" >> "$filename"

    echo "Created file: $filename"
done

echo "Successfully created 3 files"
