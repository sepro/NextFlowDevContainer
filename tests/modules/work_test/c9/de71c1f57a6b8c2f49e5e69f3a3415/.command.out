Created file: file_P0XMvlmLSers.txt
Created file: file_2q6NlFvXOUeQ.txt
Created file: file_VGrVkviTzOZC.txt
Successfully created 3 files
